README.md

```diff
@@ -135,22 +94,16 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
     > - `WORKFLOW_NAMESPACE`: Default value is set to `sonataflow-infra`.
     > - `K8S_CLUSTER_URL`: The URL of the Kubernetes cluster is obtained dynamically using `oc whoami --show-server`.
     > - `K8S_CLUSTER_TOKEN`: The value is obtained dynamically based on the provided namespace and service account.
-    > - `GITHUB_TOKEN`: This value is prompted from the user during script execution and is not predefined.
-    > - `ARGOCD_NAMESPACE`: Default value is set to `orchestrator-gitops`. The script validates that there is an ArgoCD instance running in the given namespace, else it will exit with failure.
-    > - `ARGOCD_URL`: This value is dynamically obtained based on the first ArgoCD instance available.
+    > - `GITHUB_TOKEN`: This value is prompted from the user during script execution and is not predefined. Empty if none is available.
+    > - `ARGOCD_NAMESPACE`: Default value is set to `orchestrator-gitops`. The script validates that there is an ArgoCD instance running in the given namespace, empty if none is available.
+    > - `ARGOCD_URL`: This value is dynamically obtained based on the first ArgoCD instance available. Empty if none is available.
     > - `ARGOCD_USERNAME`: Default value is set to `admin`.
-    > - `ARGOCD_PASSWORD`: This value is dynamically obtained based on the first ArgoCD instance available.
-
-9. Switch the context to the orchestrator namespace
-
-   ```console
-   oc project orchestrator
-   ```
+    > - `ARGOCD_PASSWORD`: This value is dynamically obtained based on the first ArgoCD instance available. Empty if none is available.
 
-   Install the orchestrator Helm chart:
+9.  Install the orchestrator Helm chart:
 
     ```console
-    helm upgrade -i orchestrator orchestrator --set argocd.enabled=true --set tekton.enabled=true
+    helm upgrade -i orchestrator orchestrator --set argocd.enabled=true --set tekton.enabled=true -n orchestrator
```

Comment:
This command won't work, it's for local setup. For installing by chart reference the command format is:
helm install <installation> <repo>/<chart>

Also no need to pass the namespace, step 3 creates the namespace and sets it as the current one
The correct command is

helm upgrade -i orchestrator orchestrator/orchestrator --set argocd.enabled=true --set tekton.enabled=true

Reply:
I'll update the reference to use the repo chart as you suggest. But for the namespace, I feel like having it is a benefit in case the command is copied over from a context that is not currently in the `orchestrator` namespace. Having the namespace as part of the command ensures that the chart will always be created in the expected namespace.

I understand that these commands are supposed to run sequentially and in order, but by adding the namespace we can anticipate the command being run in other scenarios.